src/index.ts

```diff
@@ -24,6 +24,39 @@ type LooseNode = Node & {
   value: any;
 };
 
+const generateHeaders = async () => {
+  await writeFile(
+    ".vercel/output/static/_headers",
+    `\n/_next/static/*\n  Cache-Control: public,max-age=31536000,immutable\n`,
+    {
+      // in case someone configured redirects already, append to the end
+      flag: "a",
+    }
+  );
+};
+
+const generateRoutes = async () => {
+  try {
+    await writeFile(
+      ".vercel/output/static/_routes.json",
+      JSON.stringify({
+        version: 1,
+        include: ["/*"],
+        exclude: ["/_next/static/*"],
```

Comment:
[Contrary to the comment above about `_headers`](https://github.com/cloudflare/next-on-pages/pull/50/files#r1134719235), I think it's fine to exclude everything after `/_next/static/` here :)